plonk/src/transcript/rescue.rs

```diff
@@ -93,27 +93,31 @@ where
         Ok(())
     }
 
-    /// Append the message to the transcript. `_label` is omitted for
-    /// efficiency.
-    fn append_message(&mut self, _label: &'static [u8], msg: &[u8]) -> Result<(), PlonkError> {
-        // We remove the labels for better efficiency
-
-        let mut f = bytes_to_field_elements(&msg);
-        self.transcript.append(&mut f);
+    /// Append the message to the transcript.
+    /// TODO(Chengyu): fix the bug here, current design brings trouble to the
```

Comment:
I updated the gadget implementation for rescue transcript. However, I encountered trouble passing the tests. The reason is described here.
I would also suggest remove this from the current release plan. 
@alxiong @chancharles92

Reply:
Agree, we can include this change in a patch release afterwards.